HW_8/logger.py

```diff
@@ -0,0 +1,9 @@
+from datetime import datetime
+
+
+def log(func):
+    def wapper(*args):
+        with open('log.csv', 'a', encoding='utf-8') as log_file:
+            log_file.write(f'{datetime.now()} - запущена функция {func.__name__} {f"с аргументами {args}" if args else "без аргументов"} ({func.__doc__})\n')
```

Comment:
У вас ни одна функция строку с документацией не имеет, а значит ее запрашивать не нужно.
```suggestion
            log_file.write(f'{datetime.now()} - запущена функция {func.__name__} {f"с аргументами {args}" if args else "без аргументов"}\n')
```